source/common/grpc/async_client_manager_impl.h

```diff
@@ -1,5 +1,9 @@
 #pragma once
 
+#include <bits/stdint-uintn.h>
```

Comment:
this doesn't look like a header you should need to manually include. Maybe just `#include <stdint.h>` or better, `#include <cstdint>`

Reply:
It is not included manually... Something wrong with bazel or formatter, will clean up.